nx.json

```diff
@@ -31,6 +31,10 @@
       "inputs": ["default", "^production"],
       "dependsOn": ["^build"]
     },
+    "build-tests": {
+      "inputs": ["default", "^production"],
+      "dependsOn": ["build", "^build"]
+    },
```

Comment:
this means that `build-tests` depends on packages `build` and dependencies `build` - this was missing to make `yarn build-tests` work at top level.